.ci/run.sh

```diff
@@ -9,10 +9,27 @@ set -e
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 
+install_docker() {
+	docker_version=${docker_version:-19.03}
```

Comment:
is this version of docker required only in this repo? why don't change in the tests repo?

Reply:
yeah, I find this is necessary only in agent, also change in tests repo won't work the version is available for 18.06 for now.

Reply:
This might work today, but what about when Docker 19.03 is no longer available? It would be much safer to check the installed version of docker and if it is "< 19.03", upgrade it.

But we store these sorts of version details in the versions database:

- https://github.com/kata-containers/runtime/blob/master/versions.yaml#L211

And if it is architecture-specific, add an `aarch64` block like here:

- https://github.com/kata-containers/runtime/blob/master/versions.yaml#L101..L106